### PR TITLE
add task

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_product-cards/)
[TEST REPORT LINK](https://vmisai.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by sTorba24 — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 1]
```diff
@@ -4,10 +4,47 @@
...
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__review-count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-label">Price:</div>
+        <div class="card__price-total">$2,199</div>
+      </div>
```
Comment (loralevitska) [inline src/index.html:41]: Use `p` tag for text according semantic 
```suggestion
        <p class="card__review-count">Reviews: 5</p>
      </div>

      <div class="card__price">
        <p class="card__price-label">Price:</p>
        <p class="card__price-total">$2,199</p>
      </div>
```

File: src/index.html [hunk 2]
```diff
@@ -4,10 +4,47 @@
...
+
+      <a
+        data-qa="hover"
+        class="card__buy"
+        href="#"
+      >Buy</a>
```
Comment (loralevitska) [inline src/index.html:47]:
```suggestion
      <a
        data-qa="hover"
        class="card__buy"
        href="#"
      >
        Buy
      </a>
```

File: src/styles/card.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
+$main-color: #00acdc;
+$white-color: #fff;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```
Comment (loralevitska) [inline src/styles/card.scss:9]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by loralevitska — CHANGES_REQUESTED** [review]
A little more and you will be on target)

- fix stars block. It should be always 5 stars in the row active/non-active
image

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**